### ISSUE TITLE
How to add a custom class to a pretrained model

### ISSUE THREAD

**Issue by Zerod159** [facebookresearch/detectron2#411]
Hi, in the Train a custom dataset section of the [Detectron2 notebook](https://colab.research.google.com/drive/16jcaJoc6bCFAQ96jDe2HwtXj7BMD_-m5#scrollTo=b2bjrfb2LDeo) is an example on hot to train a model for balloons. 

Is it possible to "add" the class and keep the existing ones?

**Comment by ppwwyyxx** [comment]
I suppose it's doable but it's unclear what you mean mathematically by "adding" a class and "keeping" the existing ones.

**Comment by Zerod159** [comment]
How do i make this clear...?

When I do the Balloon example in the notebook, do I get a model that can only detect balloons? 
But the original model (mask_rcnn_R_50_FPN_3x can detect) can detect all the classes from the COCO dataset. I want a model that can detect all the COCO object-types AND add my custom class (for example Balloons).

I hope this is a bit more clear.

**Comment by ppwwyyxx** [comment]
Then you can train it on a dataset that has both coco data and your own data (e.g. the balloons), right?

**Comment by Zerod159** [comment]
> "Because it is not a common task people do and there isn't an off-the-shelf solution for it."

Ok, I guess this part of your answer helps me the most. I thought maybe this would exist.

> "Then you can train it on a dataset that has both coco data and your own data (e.g. the balloons)"

That would mean a big computing job, right? I hoped I could avoid training a model for weeks.

**Comment by bamwani** [comment]
I have extra data for "cell phone" class. Can I finetune the "cell phone" class? and while fine tuning, my main objective is to remove all the other classes. I don't want my model to have any other class.

JUST LIKE BALLOON, but for CELL PHONE. I'm expecting that even with 300 image data, model should give almost same accuracy for cell phone as a pre-trained model.

is this possible? how?
